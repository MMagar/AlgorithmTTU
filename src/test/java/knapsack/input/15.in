330
13 23
14 24
13 23
12 22
10 20
13 23
11 21
12 22
11 21
14 24
14 24
14 24
13 23
13 23
12 22
