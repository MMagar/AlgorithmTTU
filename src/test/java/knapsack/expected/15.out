179 319
14 24
14 24
14 24
14 24
13 23
13 23
13 23
13 23
13 23
12 22
12 22
12 22
11 21
11 21
